grpc/src/main/proto/model/dynamicforkjointask.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package conductor.proto;
+
+import "google/protobuf/struct.proto";
+
+option java_package = "com.netflix.conductor.proto";
+option java_outer_classname = "DynamicForkJoinTaskPb";
+option go_package = "github.com/netflix/conductor/client/gogrpc/conductor/model";
+
+message DynamicForkJoinTask {
```

Comment:
What is the reason for one proto definition per message?

Reply:
We found this was the easiest way to keep the generated code consistent and properly namespaced.